src/installers/helm.ts

```diff
@@ -145,7 +145,7 @@ error: E_COMMAND_FAILED`)
 
   async updateCheHelmChartDependencies(cacheDir: string, execTimeout= 120000) {
     const destDir = path.join(cacheDir, '/templates/kubernetes/helm/che/')
-    await execa.shell(`helm dependencies update --skip-refresh ${destDir}`, { timeout: execTimeout })
+    await execa.shell(`helm dependencies update --skip-refresh \"${destDir}`, { timeout: execTimeout })
```

Comment:
```suggestion
    await execa.shell(`"helm" "dependencies" "update" "--skip-refresh" "${destDir}"`, { timeout: execTimeout })
```

Reply:
this is not working:

```
    × Updating Helm Chart dependencies
      → Error: could not find C:\Users\Aurelien Pupier\AppData\Local\chectl\templates\kubernetes\helm\che": GetFileAttributesEx C:\Users\Aurelien Pupier\Ap
p…
      Deploying Che Helm Chart
Error: Command failed: C:/WINDOWS/system32/cmd.exe /q /s /c ""helm" "dependencies" "update" "--skip-refresh" "C:/Users/Aurelien Pupier/AppData/Local/chectl/templates/kubernetes/helm/che/""
Error: could not find C:/Users/Aurelien Pupier/AppData/Local/chectl/templates/kubernetes/helm/che": GetFileAttributesEx C:/Users/Aurelien Pupier/AppData/Local/chectl/templates/kubernetes/helm/che": The filename, directory name, or volume label syntax is incorrect.
    at makeError (C:/git/chectl/node_modules/execa/index.js:174:9)
    at Promise.all.then.arr (C:/git/chectl/node_modules/execa/index.js:278:16)
```

Reply:
was there any specifc reasons to use execa;shell instead of execa directly?
It is working using `await execa('helm', ['dependencies', 'update', '--skip-refresh', destDir], { timeout: execTimeout, reject: false})` (which seems a nice solution)